Tasks/google-play-release/GooglePlay.ts

```diff
@@ -2,7 +2,7 @@ import fs = require('fs');
 import path = require('path');
```

Comment:
The reason that was there was for older build agent support on old TFS servers I believe. I agree it looks odd but I would suggest don't remove it as yet.